modules/sdk-core/src/account-lib/mpc/tss/ecdsa/ecdsa.ts

```diff
@@ -278,25 +324,60 @@ export default class Ecdsa {
       Ecdsa.curve.scalarInvert(d),
     ].reduce(Ecdsa.curve.scalarMult);
 
+    const { ntilde: ntildea, h1: h1a, h2: h2a } = xShare as XShareWithNTilde;
+
     const signers: SignShareRT = {
       wShare: {
         i: xShare.i,
         l: xShare.l,
         m: xShare.m,
         n: xShare.n,
         y: xShare.y,
+        ntilde: ntildea!,
```

Comment:
These three `!s` should not be necessary any more.

Reply:
Done